includes/Traits/Find_Readme.php

```diff
@@ -27,12 +27,15 @@ protected function filter_files_for_readme( array $files, $plugin_relative_path
 		// Find the readme file.
 		$readme_list = preg_grep( '/readme\.(txt|md)$/i', $files );
 
+		$plugin_relative_path = rtrim( $plugin_relative_path, '/' );
+
 		// Filter the readme files located at root.
 		$potential_readme_files = array_filter(
 			$readme_list,
 			function ( $file ) use ( $plugin_relative_path ) {
 				$file = str_replace( $plugin_relative_path, '', $file );
-				return ! str_contains( $file, '/' );
+				$file = ltrim( $file, '/\\' );
+				return ! ( str_contains( $file, '/' ) || str_contains( $file, '\\' ) );
```

Comment:
Could be simplified to:
```suggestion
				return ! str_contains( $file, '/' ) && ! str_contains( $file, '\\' );
```